mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/routes/AvtaleRoutesTest.kt

```diff
@@ -85,6 +86,117 @@ class AvtaleRoutesTest : FunSpec({
         }
     }
 
+    test("400 Bad request for autentisert kall for PUT av avtaledata dersom tiltakstypen ikke er skrudd på som master") {
```

Comment:
Kan erstatte disse tre testene med en `forAll` og eksplisistt config av enabled tiltakstype. Et utgangspunkt som må tweakes litt på for å få det til å funke:
```kotlin
        test("200 OK for autentisert kall for PUT av avtaledata når tiltakstype er enabled") {
            val avtaleSkrivRolle = AdGruppeNavAnsattRolleMapping(UUID.randomUUID(), NavAnsattRolle.AVTALER_SKRIV)
            val tiltaksadministrasjonGenerellRolle = AdGruppeNavAnsattRolleMapping(UUID.randomUUID(), NavAnsattRolle.TILTAKADMINISTRASJON_GENERELL)
            val engine = createMockEngine(
                "/brreg/enheter/${AvtaleFixtures.avtaleRequest.leverandorOrganisasjonsnummer}" to {
                    respondJson(BrregEnhet(organisasjonsnummer = "123456789", navn = "Testvirksomhet"))
                },
                "/brreg/underenheter" to {
                    respondJson(BrregEmbeddedUnderenheter(_embedded = BrregUnderenheter(underenheter = emptyList())))
                },
            )
            val config = createTestApplicationConfig().copy(
                auth = createAuthConfig(oauth, roles = listOf(avtaleSkrivRolle, tiltaksadministrasjonGenerellRolle)),
                engine = engine,
                database = databaseConfig,
                
                // sett INDOPPFAG som enabled tiltakstype
            )
            withTestApplication(config) {
                val client = createClient {
                    install(ContentNegotiation) {
                        json()
                    }
                }

                forAll(
                    row(TiltakstypeFixtures.VTA, HttpStatusCode.OK),
                    row(TiltakstypeFixtures.AFT, HttpStatusCode.OK),
                    row(TiltakstypeFixtures.Oppfolging, HttpStatusCode.OK),
                    row(TiltakstypeFixtures.Arbeidstrening, HttpStatusCode.BadRequest),
                ) { tiltakstype, status ->
                    val response = client.put("/api/v1/internal/avtaler") {
                        val claims = mapOf(
                            "NAVident" to "ABC123",
                            "groups" to listOf(avtaleSkrivRolle.adGruppeId, tiltaksadministrasjonGenerellRolle.adGruppeId),
                        )
                        bearerAuth(
                            oauth.issueToken(claims = claims).serialize(),
                        )
                        contentType(ContentType.Application.Json)
                        setBody(AvtaleFixtures.avtaleRequest.copy(navEnheter = listOf(NavEnhetFixtures.Oslo.enhetsnummer), tiltakstypeId = tiltakstype.id))
                    }
                    response.status shouldBe status
                }
            }
        }
```